### ISSUE TITLE
Bug in invalid values of errors argument in pd.to_numeric

### ISSUE THREAD

**Issue by charlesdong1991** [pandas-dev/pandas#27249]
#### Code Sample, a copy-pastable example if possible

```python
s = pd.Series(['apple', '1.0', '2', -3])

# both code work
pd.to_numeric(s, errors='abc')
pd.to_numeric(s, errors='coerce')
```
#### Problem description
If there is a typo or any value assigned to `errors` argument other than `raise` nor `ignore`, it will be considered as `coerce`.
I track down to the source code, it turns out pandas doesn't check valid values for this argument.

#### Expected Output
An error should be raised.

**Comment by jreback** [comment]
can u confirm this in master?

**Comment by charlesdong1991** [comment]
@jreback ahh, yeah, sorry, the bug was fixed in 0.25rc, i am using 0.24x
i will close the issue.
